AWSome.Chatter/Transactions/KinesisReceiver.cs

```diff
@@ -36,6 +36,7 @@ public static List<Record> GetRecordsFromKinesisQueue(DateTime fromDateTime)
         /// </summary>
         /// <param name="kinesisClient">The Kinesis client object</param>
         /// <param name="describeResponse">The stream response descriptor</param>
+        /// <param name="fromDateTime"></param>
```

Comment:
hmmm looking back on this I think the name of fromDateTime is a little confusing - is this getting the messages after a specific DateTime? Perhaps we should rename it? What do you think?

Reply:
It looks to be the timestamp for the first record to fetch an iterator for.

Does timestamp seem like the better name? Or firstTimestamp? Will leave as is for now until you let me know what you think :)